.github/workflows/main.yml

```diff
@@ -154,6 +160,9 @@ jobs:
   # 4. Adjust website restrictions for Firebase Key "Sharezone Web Key".
   web-preview:
     runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write # for FirebaseExtended/action-hosting-deploy to comment on PRs
+      checks: write # for FirebaseExtended/action-hosting-deploy
```

Comment:
Couldn't find what the `checks` permissions is exactly for. Can you add it to the comment next to it?

Reply:
The thing is that I also don't know why it's needed but without it's not posting the comment. It also needs to be `write`. `read` is not working.

Reply:
Added the comment

> for FirebaseExtended/action-hosting-deploy to comment on PRs (without write permissions for checks the action doesn't post a comment to the PR)